install/Makefile

```diff
@@ -142,6 +143,16 @@ export HUGO_VERSION ?= 0.45.1
 hugo: hugo-$(OS)
 	@exit 0
 
+export SHFMT_VERSION ?= 2.5.1
+# Releases: https://github.com/mvdan/sh/releases
+## Install shfmt - shell parser, formatter and interpreter
+shfmt: sh
+	@exit 0
+
+# For overriding $@
+sh: 
```

Comment:
Why does `shfmt` call `sh`? Doesn't look necessary. Also, `sh` is misleading since that collides with the comon name for the POSIX shell.

Reply:
Consolidate `shfmt` with `sh`. Keep `shfmt` target